config/config.go

```diff
@@ -98,6 +105,39 @@ func (c standardConfigManager) Get(key string) ([]byte, error) {
 	return value, err
 }
 
+// List retrieves and decodes all secconf value stored under key.
+func (c configManager) List(key string) (KVPairs, error) {
+	list, err := c.store.List(key)
+	retList := make(KVPairs, len(list))
+	if err != nil {
+		return nil, err
+	}
+	for i, kv := range list {
+		retList[i].Key = kv.Key
+		retList[i].Value, err = secconf.Decode(kv.Value, bytes.NewBuffer(c.keystore))
+		if err != nil {
+			return nil, err
+		}
+	}
+	return retList, nil
+}
+
+// List retrieves all values under key.
+// convenience function, no additional value provided over
+// `etcdctl`
+func (c standardConfigManager) List(key string) (KVPairs, error) {
+	list, err := c.store.List(key)
+	retList := make(KVPairs, len(list))
+	if err != nil {
+		return nil, err
+	}
+	for i, kv := range list {
+		retList[i].Key = kv.Key
```

Comment:
This is to prevent leaking the backend.KVPairs type out of the library, is there a nicer way to do this?